PrestaShop/PrestaShop
# Migration of  "Design -> pages -> cms page listing"

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This pr creates a a cms list in **admin-dev/index.php/improve/design/cms-pages/**
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10565
| How to test?  | See below

## How to test

This page has 2 listings. The top listing is already OK and this page enables the bottom listing. Everything should work except known issues:
- the **create new cms page button is missing**
- the **edit button will throw an error since it is not implemented yet**
-  **position stuff does not work**
- **no drag & drop support yet**




---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/12981)
